assets/js/modules/analytics/components/settings/SettingsForm.stories.js

```diff
@@ -37,13 +38,13 @@ function Template() {
 			<div className="googlesitekit-settings-module googlesitekit-settings-module--active googlesitekit-settings-module--analytics">
 				<div className="googlesitekit-setup-module">
 					<div className="googlesitekit-settings-module__content googlesitekit-settings-module__content--open">
-						<div className="mdc-layout-grid">
+						<Grid>
 							<div className="mdc-layout-inner">
```

Comment:
Looks like this class name is a typo and it should be `mdc-layout-grid__inner` instead of just `mdc-layout-inner`. Thus we need to replace it with `<Row>`. The same applies to all 4 files where this class name is present.

Reply:
Thanks, good spot!

Reply:
Done :+1: